pgcli/main.py

```diff
@@ -506,24 +503,25 @@ def handle_editor_command(self, cli, document):
         "select * from \e"<enter> to edit it in vim, then come
         back to the prompt with the edited query "select * from
         blah where q = 'abc'\e" to edit it again.
-        :param cli: CommandLineInterface
-        :param document: Document
+        :param text: Document
         :return: Document
         """
+        app = self.prompt_app.app
+
         # FIXME: using application.pre_run_callables like this here is not the best solution.
         # It's internal api of prompt_toolkit that may change. This was added to fix #668.
         # We may find a better way to do it in the future.
-        saved_callables = cli.application.pre_run_callables
+        saved_callables = app.pre_run_callables
```

Comment:
@jonathanslenders removed this workaround in the mycli branch for python prompt toolkit, is this fixed in prompt toolkit 2.0?